protocol/avdl/keybase1/teams.avdl

```diff
@@ -1033,7 +1033,7 @@ protocol teams {
    * If refreshers are non-empty, then force a refresh of the cache if the requirements
    * of the refreshers aren't met.
```

Comment:
"force a refresh of the cache if the requirements"
unless doing the offline thing, then might return old stuff